cockroachcloud/network-authorization.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Network Authorization
+summary: Learn about the network authorization features for CockroachCloud CockroachDB clusters.
+toc: true
+---
+
+To prevent denial-of-service and brute force password attacks, CockroachCloud requires you to authorize networks that can access your cluster by [allowlisting the public IP addresses](#ip-allowlisting) for your application. Optionally, you can [set up Virtual Private Cloud (VPC) peering](#vpc-peering) for your cluster for enhanced network security and lower network latency.
+
+## IP allowlisting
+
+Authorize your application server’s network and your local machine’s network by [adding their public IP addresses (in the CIDR format) to the CockroachCloud cluster's allowlist](connect-to-your-cluster.html#step-1-authorize-your-network). If you change your location, you will need to authorize the new location’s network, else the connection from that network will be rejected.
+
+- In a development environment, you need to authorize your application server’s network and your local machine’s network. If you change your location, you need to authorize the new location’s network, or else the connection from that network will be rejected.
```

Comment:
This is saying the same thing as the previous paragraph.  Maybe bears repeating?

Reply:
I meant the first para to be an intro to the section and then have the bullet points specify the allowlist requirements for prod and dev environments. Totally open to suggestions :)